agent/consul/authmethod/kubeauth/k8s.go

```diff
@@ -132,24 +135,24 @@ func (v *Validator) ValidateLogin(loginToken string) (map[string]string, *struct
 	})
 
 	if err != nil {
-		return nil, nil, err
+		return nil, err
 	} else if trResp.Status.Error != "" {
-		return nil, nil, fmt.Errorf("lookup failed: %s", trResp.Status.Error)
+		return nil, fmt.Errorf("lookup failed: %s", trResp.Status.Error)
 	}
 
 	if !trResp.Status.Authenticated {
-		return nil, nil, errors.New("lookup failed: service account jwt not valid")
+		return nil, errors.New("lookup failed: service account jwt not valid")
```

Comment:
Using errors.New() here because we may be returning this error multiple times? Or because we aren't passing in a value to be `fmt`ed with the error message?

Reply:
The difference between usage of `errors.New` and `fmt.Errorf` is very thin. In this case it's the latter (nothing variable about the error).